mlflow/pyfunc/model.py

```diff
@@ -299,10 +271,10 @@ def predict_stream(
                 inference.
 
         Returns:
-            An iterator over :py:class:`ChatResponse <mlflow.types.llm.ChatResponse>` object
-            containing the model's response(s), as well as other metadata.
+            A generator over :py:class:`ChatCompletionChunk <mlflow.types.llm.ChatCompletionChunk>`
+            object containing the model's response(s), as well as other metadata.
         """
-        yield self.predict(context, messages, params)
+        raise NotImplementedError()
```

Comment:
what does the failure look like in model serving if u call /invocations with `stream: True` for a model that does not have an implemented streaming interface?

Reply:
We'll throw a 400 bad request with the stack trace